model/context.go

```diff
@@ -89,7 +89,14 @@ func (ctx *Context) populateTaskBuildVersion(taskId, buildId, versionId string)
 	if len(taskId) > 0 {
 		ctx.Task, err = task.FindOne(task.ById(taskId))
 		if err != nil {
-			return "", err
+			// if no task found, see if this is an old task
+			tasks, err := task.FindOld(task.ById(taskId))
+			if err != nil {
+				return "", err
+			}
+			if len(tasks) == 1 {
+				ctx.Task = &tasks[0]
```

Comment:
What are the other tasks that can come up? Other executions?

Reply:
It shouldn't be possible for any other tasks to come up - this is a sanity check

Reply:
Maybe through an ErrorWhen(len(tasks) > 1)? otherwise lgtm

Reply:
I think this is fine because it's an id query. It's also legit to add a task.FindOldOne() helper to the packag.e